mycpp/gc_heap.h

```diff
@@ -1175,7 +1179,7 @@ class Dict : public gc_heap::Obj {
   V index_(K key) {
     int pos = position_of_key(key);
     if (pos == -1) {
-      assert(0); // NOTE(Jesse): Should we really crash if asking for a key not in a dict?
+      InvalidCodePath(); // NOTE(Jesse): Should we really crash if asking for a key not in a dict?
```

Comment:
I'm not sure exactly what we should do here -- I guess we do the minimal thing to make the shell work.  If the shell never uses it, then we never have to fix this :-P

Reply:
Agreed.  I left this comment in as more of a reminder for if we ever hit it.